src/Microsoft.ML.Onnx/OnnxContextImpl.cs

```diff
@@ -13,11 +13,12 @@ namespace Microsoft.ML.Runtime.Model.Onnx
     /// <summary>
     /// A context for defining a ONNX output.
     /// </summary>
-    internal sealed class OnnxContextImpl : OnnxContext
+    public sealed class OnnxContextImpl : OnnxContext
```

Comment:
>public [](start = 4, length = 6)

I would ask @TomFinley why he prefer to make it internal. If only reason why you want to make it public is to make it visible in tests, you can use InternalVisibileTo attribute

Reply:
Thank you. It finally works with `[assembly: InternalsVisibleTo("Microsoft.ML.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010015c01ae1f50e8cc09ba9eac9147cf8fd9fce2cfe9f8dce4f7301c4132ca9fb50ce8cbf1df4dc18dd4d210e4345c744ecb3365ed327efdbc
52603faa5e21daa11234c8c4a73e51f03bf192544581ebe107adee3a34928e39d04e524a9ce729d5090bfd7dad9d10c722c0def9ccc08ff0a03790e48bcd1f9b6c476063e1966a1c4")]`